Makefile

```diff
@@ -110,7 +118,9 @@ install-crds: manifests-controllers kustomize ## Install CRDs into the K8s clust
 uninstall-crds: manifests-controllers kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build controllers/config/crd | kubectl delete -f -
 
-deploy: deploy-controllers deploy-api
+deploy: install-crds deploy-controllers deploy-api
+
+deploy-kind: install-crds deploy-controllers deploy-api-kind-auth
```

Comment:
I made a separate PR for these changes. Nothing in particular will need to be done, but if you are already in here fixing the kustomization file changes, you might consider also removing this.

Reply:
I remove them in commit https://github.com/cloudfoundry/cf-k8s-controllers/pull/248/commits/3a27a36419ea9c5882d6fa330e393fe9686362d6